app/src/main/java/com/mstudio/superheroarch/CharacterDao.kt

```diff
@@ -0,0 +1,15 @@
+package com.mstudio.superheroarch
+
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+
+@Dao
+interface CharacterDao {
+    @Insert(onConflict = OnConflictStrategy.REPLACE)
+    suspend fun insertAll(vararg characters: Character)
```

Comment:
we talked specifically about this vararg, change it for a list

Reply:
Fixed in next commit